orocos_kdl/src/chainidsolver_vereshchagin.cpp

```diff
@@ -25,12 +25,16 @@
 #include "frames_io.hpp"
 #include "utilities/svd_eigen_HH.hpp"
 
+/**
+* For more details on this solver, see the documentation in 
+* "chainhdsolver_vereshchagin_doc.md"
+*/
 
 namespace KDL
 {
 using namespace Eigen;
 
-ChainIdSolver_Vereshchagin::ChainIdSolver_Vereshchagin(const Chain& chain_, Twist root_acc, unsigned int _nc) :
+ChainHdSolver_Vereshchagin::ChainHdSolver_Vereshchagin(const Chain& chain_, const Twist root_acc, const unsigned int _nc):
```

Comment:
Change the class name breaks the API. The change is so small that it is not worth it.

Use const reference.

Keep space to colon

Reply:
Regarding the change of class name: The name was changed because the "id" part in the class name was misleading to users. However, the name of the file still remained "chainidsolver_vereshchagin" due to the necessity of keeping the track of the changes made in the file. I did not want to change the file name before collaborator's review. Nevertheless, I would suggest keeping the name (since it describes the algorithm correctly) and later on, when other points that you have outlined are corrected, I would suggest to rename the file name as well and update components of KDL that uses this file. I 
think there are not many components (other files) that call this class; from the first search, it seems that only the solvertest files import this class.

Reply:
Regrading the const reference: what exactly do you mean by that? The arguments under the constructor are already under const.